### PR TITLE
All games, mini-programs, and runtime platforms with onWindowResize need to listen to it.

### PR DESCRIPTION
**Pull request by bofeng-song**
Re: #

### Changelog

*

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: pal/screen-adapter/minigame/screen-adapter.ts [hunk 1]
```diff
@@ -156,11 +156,9 @@ class ScreenAdapter extends EventTarget {
     constructor () {
         super();
         // TODO: onResize or onOrientationChange is not supported well
-        if (WECHAT || WECHAT_MINI_PROGRAM || RUNTIME_BASED) {
-            minigame.onWindowResize?.(() => {
-                this.emit('window-resize', this.windowSize.width, this.windowSize.height);
-            });
-        }
+        minigame.onWindowResize?.(() => {
```
Comment (minggo) [inline pal/screen-adapter/minigame/screen-adapter.ts:158]: Does all mini game platforms support resize event well?

Comment (bofeng-song) [reply pal/screen-adapter/minigame/screen-adapter.ts:158]: If any platform does not support it well, it should be a hidden problem that needs to be fixed

Comment (minggo) [reply pal/screen-adapter/minigame/screen-adapter.ts:158]: Any issue for it?

**Review by PPpro — APPROVED** [review]

File: pal/screen-adapter/minigame/screen-adapter.ts [hunk 2]
```diff
@@ -51,6 +51,7 @@ try {
         }
     }
 } catch (e) {
+    // eslint-disable-next-line no-console
```
Comment (minggo) [inline pal/screen-adapter/minigame/screen-adapter.ts]: Don't do it like this. It just avoid error. I fixed in develop branch, so don't have to do in v3.8.1.

File: pal/screen-adapter/minigame/screen-adapter.ts [hunk 3]
```diff
@@ -110,6 +111,7 @@ class ScreenAdapter extends EventTarget {
         return minigame.orientation;
     }
     public set orientation (value: Orientation) {
+        // eslint-disable-next-line no-console
```
Comment (minggo) [inline pal/screen-adapter/minigame/screen-adapter.ts]: The same as above.

File: pal/screen-adapter/minigame/screen-adapter.ts [hunk 4]
```diff
@@ -156,11 +158,9 @@ class ScreenAdapter extends EventTarget {
     constructor () {
         super();
         // TODO: onResize or onOrientationChange is not supported well
```
Comment (minggo) [inline pal/screen-adapter/minigame/screen-adapter.ts]: Should remove the comment now?

**Review by minggo — APPROVED** [review]

**Pull request merged**